restapi/src/main/java/io/stargate/web/docsapi/resources/DocumentResource.java

```diff
@@ -0,0 +1,334 @@
+package io.stargate.web.docsapi.resources;
+
+import com.datastax.oss.driver.api.core.NoNodeAvailableException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.databind.node.TextNode;
+import io.stargate.auth.UnauthorizedException;
+import io.stargate.web.docsapi.dao.DocumentDB;
+import io.stargate.web.docsapi.exception.DocumentAPIRequestException;
+import io.stargate.web.docsapi.service.DocumentService;
+import io.stargate.web.docsapi.service.filter.FilterCondition;
+import io.stargate.web.resources.Db;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.Callable;
+import java.util.stream.Collectors;
+import javax.inject.Inject;
+import javax.ws.rs.*;
+import javax.ws.rs.core.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Path("/v1/")
```

Comment:
I thought this was going under v2?

Reply:
`DocumentResource` is the old v1 resource; `DocumentV2Resource` is the v2 stuff

Should I not have ported this over at all? I can delete